src/feedback.tsx

```diff
@@ -324,3 +379,62 @@ export const InfoFeedbackScreen: React.FC<FeedbackProps> = (props) => {
     const hasIcon = skinName !== VIVO_SKIN;
     return <FeedbackScreen {...props} icon={hasIcon ? <IcnInfo /> : undefined} />;
 };
+
+export const SuccessFeedback: React.FC<AssetFeedbackProps> = ({
+    title,
+    description,
+    children,
+    primaryButton,
+    secondaryButton,
+    link,
+    imageUrl,
+    imageFit,
+    dataAttributes,
+}) => {
+    useHapticFeedback('success');
+    const {isTabletOrSmaller} = useScreenSize();
+    const {skinName} = useTheme();
+    const {colors} = useTheme();
+    const hasButtons = checkHasButtons({primaryButton, secondaryButton, link});
+
+    const classes = useStyles({
+        isInverse: true,
+        animateText,
+        primaryButton,
+        imageUrl,
+        imageFit,
+        hasButtons,
+    });
+    const icon = skinName === VIVO_SKIN ? <IconSuccessVivo /> : <IcnSuccess />;
+    const feedbackBody = renderFeedbackBody({icon, title, description, children}, classes, colors);
+    const inlineFeedbackBody = renderInlineFeedbackBody(
+        feedbackBody,
+        {
+            primaryButton,
+            secondaryButton,
+            link,
+        },
+        classes
+    );
+
+    return (
+        <ThemeVariant isInverse>
+            {isTabletOrSmaller ? (
+                <ResponsiveLayout className={classes.background}>
```

Comment:
ResponsiveLayout included here to implement this:
![image](https://user-images.githubusercontent.com/681159/134879908-7a0ddba7-de44-4d8e-8a63-f520d42c10b5.png)